server/chat-plugins/friends.ts

```diff
@@ -268,8 +272,8 @@ export const commands: Chat.ChatCommands = {
 				sendPM(`/uhtmlchange sent,`, targetUser.id);
```

Comment:
Is there a need to wait for the database before sending UI updates?

Reply:
That's why I would prefer to run them at the end of commands rather than in handlers - so we can await it and ensure the cache is updated (so we can avoid issues).

Reply:
In case i misunderstood, we wait for the database because verification that we can actually perform this action is mostly done there.